### PR TITLE
Add basic KUTTL tests for neutron-operator

### PR DESCRIPTION
**Pull request by cescgina**
These tests cover basic operations: deployment, scale up, scale down and clean up.

The tests use code from the install_yamls repo to reuse some operations like
the deployment of the neutron operator, as well as the cleanup from previous runs.

Test are thought to be run from the `install_yamls` repo, see the following
PR: https://github.com/openstack-k8s-operators/install_yamls/pull/99

There are three ways to run the tests:
1) Run `make neutron_kuttl`, this will only work if you want to run the test
from the master branch.
2) Run `make neutron_kuttl NEUTRON_REPO=ovs-operator._repo_url
NEUTRON_BRANCH=branch_name` this is useful if you want to run the tests from
a fork
3) Run `make neutron_kuttl
NEUTRON_KUTTL_CONF=/path/to/neutron-operator/kuttl-test.yaml
NEUTRON_KUTTL_DIR=/path/to/neutron-operator/tests/kuttl/tests/` this is
useful to run local changes to the tests

### CODE REVIEW

File: kuttl-test.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# 3. CLI user has access to $KUBECONFIG
+# 4. The environment variable INSTALL_YAMLS is set to the the path of the
+#    install_yamls repo
+
+apiVersion: kuttl.dev/v1beta1
+kind: TestSuite
+reportFormat: JSON
+reportName: kuttl-test-neutron
+namespace: openstack
+timeout: 180
```
Comment (marios) [inline kuttl-test.yaml:22]: adding link incase useful for other reviewers

these config attributes are described there https://kuttl.dev/docs/testing/reference.html#testsuite
e.g. timeout is in seconds

**Review by marios — COMMENTED** [review]
lgtm didn't spot something

File: kuttl-test.yaml (hunk 1 above)
Comment (marios) [reply kuttl-test.yaml:22]: though perhaps the subject of the pull request can be updated

it is the same as the install_yamls one https://github.com/openstack-k8s-operators/install_yamls/pull/99 where it makes more sense

Comment (cescgina) [reply kuttl-test.yaml:22]: Ah, you're right, I've mixed them up, since I created one after the other, I'll update it, thanks

**Review by abays — APPROVED** [review]
/lgtm

**Pull request merged**